ControlFlow/Program.cs

```diff
@@ -14,23 +14,56 @@ public static class ControlFlow
         // (This logic is used a lot in applications where values entered into input boxes need to
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
+            
         {
-            return default;
+            if (number <= 1 || number >= 10)
+            {
+                return "Invalid";
+            }
+
+            else
+            {
+                return "Valid";
+            }
+        
+        
         }
 
+
         // 1.2 - Write a method which takes two integers and returns the maximum of the two. If they
         // are the same, return either one.
         public static int Maximum(int number1, int number2)
         {
-            return default;
+            var result = 0;
+            if (number1 > number2)
+            {
+                result = number1;
+
+            }
+            else
+            {
+                result= number2;
+            }
+            return result;
         }
 
         // 1.3 - Write a method which takes two integers, the width and height of an image. Then
         // return a string to tell if the image is "Landscape" or "Portrait". Return "Square" if
         // they are equal. Assume non-negative values.
         public static string LandscapeOrPortrait(int width, int height)
         {
-            return default;
+            if (width > height)
+            {
+                return "Landscape";
+            }
+            else if (width < height)
+            {
+                return "Portrait";
+            }
+            else
+            {
+                return "Square";
+            }
         }
```

Comment:
In the future try to reduce the number of return statements by introducing a result variable.